src/DGtal/kernel/UnorderedSetByBlock.h

```diff
@@ -30,177 +30,48 @@
 #include <boost/iterator/iterator_facade.hpp>
```

Comment:
```suggestion
#include <boost/iterator/iterator_facade.hpp>
#include <climits>
```
For `CHAR_BITS` used below.